doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -1632,3 +1632,416 @@ def write_cdb(port,cmd):
+
+
+#### 7.4 CLI Sample Output format
+```
+root@sonic:/home/cisco# show interface transceiver pm history 60sec window 1 -n asic0 Ethernet2                                                                          
+Tue Jan 31 09:25:16 UTC 2023
+PM window: 60 sec
+PM window start time: Tue Jan 31 09:24:03 UTC 2023
+Ethernet2:
+    Parameter        Unit    Min       Avg       Max       Threshold    Threshold    Threshold     Threshold    Threshold    Threshold
+                                                           High         High         Crossing      Low          Low          Crossing
+                                                           Alarm        Warning      Alert-High    Alarm        Warning      Alert-Low
+    ---------------  ------  --------  --------  --------  -----------  -----------  ------------  -----------  -----------  -----------
+    Tx Power         dBm     -8.22     -8.23     -8.24     -5.0         -6.0         False         -16.99       -16.003      False
+    Rx Total Power   dBm     -10.61    -10.62    -10.62    2.0          0.0          False         -21.0        -18.0        False
+    Rx Signal Power  dBm     -40.0     0.0       40.0      13.0         10.0         True          -18.0        -15.0        True
+    CD-short link    ps/nm   0.0       0.0       0.0       1000.0       500.0        False         -1000.0      -500.0       False
+    PDL              dB      0.5       0.6       0.6       4.0          4.0          False         0.0          0.0          False
+    OSNR             dB      36.5      36.5      36.5      99.0         99.0         False         0.0          0.0          False
+    eSNR             dB      30.5      30.5      30.5      99.0         99.0         False         0.0          0.0          False
+    CFO              MHz     54.0      70.0      121.0     3800.0       3800.0       False         -3800.0      -3800.0      False
+    DGD              ps      5.37      5.56      5.81      7.0          7.0          False         0.0          0.0          False
+    SOPMD            ps^2    0.0       0.0       0.0       655.35       655.35       False         0.0          0.0          False
+    SOP ROC          krad/s  1.0       1.0       2.0       N/A          N/A          N/A           N/A          N/A          N/A
+    Pre-FEC BER      N/A     4.58E-04  4.66E-04  5.76E-04  1.25E-02     1.10E-02     0.0           0.0          0.0          0.0
+    Post-FEC BER     N/A     0.0       0.0       0.0       1000.0       1.0          False         0.0          0.0          False
+    EVM              %       100.0     100.0     100.0     N/A          N/A          N/A           N/A          N/A          N/A
+```
+#### 7.5 High Level Design
+
+##### 7.5.1 Configurations
+1. performance monitor enable - Global CLI to enable PM on all ports. 
+Before this configuration get implemented, PM will be enabled by default on all ports
```

Comment:
@prgeor , Yes, this will future implementation. Global CLI to enable/disable PM on all ports. 
Raised a github issue :- https://github.com/sonic-net/sonic-platform-daemons/issues/402